doc/source/whatsnew/v1.0.0.rst

```diff
@@ -333,52 +329,62 @@ New repr for :class:`~pandas.arrays.IntervalArray`
 ``DataFrame.rename`` now only accepts one positional argument
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- :meth:`DataFrame.rename` would previously accept positional arguments that would lead
-  to ambiguous or undefined behavior. From pandas 1.0, only the very first argument, which
-  maps labels to their new names along the default axis, is allowed to be passed by position
-  (:issue:`29136`).
+:meth:`DataFrame.rename` would previously accept positional arguments that would lead
+to ambiguous or undefined behavior. From pandas 1.0, only the very first argument, which
+maps labels to their new names along the default axis, is allowed to be passed by position
+(:issue:`29136`).
+
+.. ipython:: python
```

Comment:
Do we need this still? Oh, is it just for the linter?

Reply:
no, there is still one case below that uses the dataframe that is an ipython block